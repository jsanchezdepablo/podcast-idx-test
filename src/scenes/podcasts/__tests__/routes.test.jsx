import React from "react";
import { shallow } from "enzyme";
import PodcastRoutes from "../routes";

jest.mock("react", () => ({
  ...jest.requireActual("react"),
  lazy: (load) => {
    return load();
  },
}));

describe("Podcasts Routes", () => {
  it("Matches snapshot", async () => {
    const wrapper = shallow(<div>{PodcastRoutes}</div>);
    expect(wrapper).toMatchSnapshot();
  });
});
