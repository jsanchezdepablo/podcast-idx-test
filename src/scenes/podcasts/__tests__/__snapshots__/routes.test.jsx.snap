// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Podcasts Routes Matches snapshot 1`] = `
<div>
  <Route
    exact={true}
    key="detail"
    path="/podcast/:podcastId"
  >
    <Component />
  </Route>
  <Route
    exact={true}
    key="episode"
    path="/podcast/:podcastId/episode/:episodeId"
  >
    <Component />
  </Route>
  <Route
    exact={true}
    key="podcasts"
    path="/"
  >
    <Component />
  </Route>
</div>
`;
